src/components/course/CourseMaterialsButton.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { Button } from '@edx/paragon';
+import { findUserEnrollmentForCourseRun } from './data/utils';
+import { CourseContext } from './CourseContextProvider';
+
+const CourseMaterialsButton = () => {
+  const {
+    state: {
+      course,
+      userEnrollments,
+    },
+  } = useContext(CourseContext);
+
+  let enrolledCourseRun;
+  let userEnrollment;
+  for (const courseRun of course.courseRuns) { // eslint-disable-line no-restricted-syntax
+    const userEnrollmentForCourseRun = findUserEnrollmentForCourseRun({
+      userEnrollments,
+      key: courseRun.key,
+    });
+    if (userEnrollmentForCourseRun) {
+      userEnrollment = userEnrollmentForCourseRun;
+      enrolledCourseRun = courseRun;
+      break;
+    }
+  }
+
+  if (!enrolledCourseRun) {
```

Comment:
nit: Since we're not using enrolledCourseRun for anything else...
```suggestion
  if (!userEnrollment) {
```